lib/backpex/filters/boolean.ex

```diff
@@ -38,6 +38,8 @@ defmodule Backpex.Filters.Boolean do
   > In addition it will add a `render` and `render_form` function in order to display the corresponding filter.
   > It will also implement the `Backpex.Filter.query` function to define a boolean query.
   """
+  use Phoenix.Component, global_prefixes: ~w(x-)
```

Comment:
Can't we `use BackpexWeb, :filter` here? `global_prefixes` are already defined there. Seems redundant in every filter module.